### PR TITLE
Payee 132

### PR DESCRIPTION
**Pull request by brendanstraughan**
- [x] Unit tests passing
- [ ] Coverage reached
- [x] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by Jamie22Butler — APPROVED** [review]
LGTM 👍

File: test/controllers/PayeControllerHistoricSpec.scala [hunk 1]
```diff
@@ -108,6 +108,23 @@ class PayeControllerHistoricSpec
       }
     }
 
+    "return ok and use last FPS(Full payment submission) for total taxable income when there are multiple same date submissions" in {
```
[the remark this answers is not in the document]
Comment (brendanstraughan) [reply test/controllers/PayeControllerHistoricSpec.scala:111]: on line 317 you can see there is an employment with an annual account that has nil for payments, this is used in a few tests. Happy with that or do you want a nil case added for multiple same date FPS submissions?

https://github.com/hmrc/tai-frontend/pull/846/files/cd61974b59e7d929620db9af900a37263887f703#diff-54b2059bde8602728a92425b9e12001594dc20a8fe417f81816b472aaf0b8743R317

[the remark this answers is not in the document]
Comment (sergi-hmrc) [reply test/controllers/PayeControllerHistoricSpec.scala:111]: you can use that one, it's fine

**Review by sergi-hmrc — APPROVED** [review]
LGTM!

**Pull request merged**